recipes/oatpp-websocket/all/test_package/CMakeLists.txt

```
@@ -7,6 +7,7 @@ conan_basic_setup()
 find_package(oatpp REQUIRED CONFIG)
 find_package(oatpp-websocket REQUIRED CONFIG)
 
-add_executable(${PROJECT_NAME} test_package.cpp FullAsyncTest.cpp)
-target_link_libraries(${PROJECT_NAME} oatpp::oatpp oatpp::oatpp-test oatpp::oatpp-websocket)
+add_executable(${PROJECT_NAME} test_package.cpp)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+# Use linking with components to properly test package_info()
+target_link_libraries(${PROJECT_NAME} oatpp::oatpp oatpp::oatpp-test oatpp::oatpp-websocket)
```

I'm wondering, does it really need to link to all the targets?
I guess it depends internally to the main `oatpp::oatpp`?
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE oatpp::oatpp-websocket)
```

---

`oatpp::oatpp` doesn't depend on `oatpp::oatpp-test`, so it was required because test_package was relying on this target. But with this major cleanup, I don't know.